lua/core/dashboard.lua

```diff
@@ -69,14 +69,22 @@ M.setup = function()
 
   vim.g.dashboard_session_directory = lvim.builtin.dashboard.session_directory
 
-  vim.cmd "let packages = len(globpath('~/.local/share/lunarvim/site/pack/packer/start', '*', 0, 1))"
+  local lv_site = "lunarvim.org"
+  local lv_path = "$HOME/.local/share/lunarvim"
```

Comment:
Should be replaced by LUNARVIM_RUNTIME_DIR once https://github.com/LunarVim/LunarVim/pull/1381 is merged

Reply:
Ty, will keep my eyes on it :)